### PR TITLE
Testing Gs

### PR DESCRIPTION
**Pull request by jessicaguo**
This PR adds a single script, Gs_testing.Rmd, which looks at 3 levels of g1 in BioCro run at 3 locations in 2010. The effect of lowered g1 is only obvious at one of three locations, Pueblo, CO. It is particularly noticeable in T/ET. 

This troubleshooting gives rise to other ideas for visualizing the effects of lowered g1 across time and space.

### CODE REVIEW

File: BioCro/DARPA/Gs_testing.Rmd [hunk 1]
```diff
@@ -0,0 +1,253 @@
+  facet_wrap(~location, scales = "free_y", ncol = 1)
```
Comment (KristinaRiemer) [inline BioCro/DARPA/Gs_testing.Rmd:1]: I think these two lines got in here accidentally? 
```suggestion

```

File: BioCro/DARPA/Gs_testing.Rmd [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+```---
```
Comment (KristinaRiemer) [inline BioCro/DARPA/Gs_testing.Rmd:2]:
```suggestion
---
```

File: BioCro/DARPA/Gs_testing.Rmd [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+library(data.table)
+```
+
+## Testing sensitivity of modeled transpiration and biomass to Gs parameters
+
+Regional runs of BioCro using the parameters from the 'high light' chamber treatment were conducted, but no differences in transpiration or biomass were found for the stomatal parameter experiment. Here, we test a range of stomatal parameter values at 3 different locations to a) isolate the sensitivity of transpiration and biomass to the stomatal slope and b) understand how this sensitivity may differ spatially due to environmental conditions. 
+
+This set of runs uses the same parameters as above for the high light treatment, except in `config.xml` the stomatal slope input value is changed from `r config$pft$photoParms$b1` to `r config_ss$pft$photoParms$b1`. 
+
+Changed Carbon Allocation Regional Runs
```
Comment (KristinaRiemer) [inline BioCro/DARPA/Gs_testing.Rmd:31]: This script is only for stomatal slope, not carbon allocation, right? 
```suggestion

```

File: BioCro/DARPA/Gs_testing.Rmd [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+
+This set of runs uses the same parameters as above for the high light treatment, except in `config.xml`
```
Comment (KristinaRiemer) [inline BioCro/DARPA/Gs_testing.Rmd:33]:
```suggestion

```

File: BioCro/DARPA/Gs_testing.Rmd [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+library(udunits2)
+library(ggplot2)
+library(data.table)
+```
+
+## Testing sensitivity of modeled transpiration and biomass to Gs parameters
+
+Regional runs of BioCro using the parameters from the 'high light' chamber treatment were conducted, but no differences in transpiration or biomass were found for the stomatal parameter experiment. Here, we test a range of stomatal parameter values at 3 different locations to a) isolate the sensitivity of transpiration and biomass to the stomatal slope and b) understand how this sensitivity may differ spatially due to environmental conditions. 
+
+This set of runs uses the same parameters as above for the high light treatment, except in `config.xml` the stomatal slope input value is changed from `r config$pft$photoParms$b1` to `r config_ss$pft$photoParms$b1`.
```
Comment (KristinaRiemer) [inline BioCro/DARPA/Gs_testing.Rmd:29]: This produces an error when knitting because there's no config object at that point in the script.

**Review by KristinaRiemer — APPROVED** [review]
I only reviewed `Gs_testing.Rmd`, with the associated config files, and everything worked when running chunks in the console (except I don't have permissions to `/data/output/biocro_regional_runs/gs_test_` so I just changed it to a local path).

File: BioCro/DARPA/Gs_testing.Rmd (hunk 1 above)
Comment (jessicaguo) [reply BioCro/DARPA/Gs_testing.Rmd:1]: Oops good catch!

File: BioCro/DARPA/Gs_testing.Rmd (hunk 2 above)
Comment (jessicaguo) [reply BioCro/DARPA/Gs_testing.Rmd:2]: Fixed

File: BioCro/DARPA/Gs_testing.Rmd (hunk 3 above)
Comment (jessicaguo) [reply BioCro/DARPA/Gs_testing.Rmd:31]: Yup, I need to edit my copy+paste jobs better

File: BioCro/DARPA/Gs_testing.Rmd (hunk 5 above)
Comment (jessicaguo) [reply BioCro/DARPA/Gs_testing.Rmd:29]: I eliminated the second section alltogether, was overlooked from the previous document.

**Pull request merged**